capstone/src/main/java/com/google/step/servlets/UserSignUpServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.google.step.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that will put the user information in Datastore.
+ */
+@WebServlet("/userSignUp")
```

Comment:
Similar to comments above, consider making this the "/user" servlet where a GET gets the user info, and a POST creates a new user.

Reply:
Ended up deleting the entire servlet due to the change of user-restaurant interaction.